ts/Stock/Indicators/RSI/RSIIndicator.ts

```diff
@@ -88,9 +88,10 @@ class RSIIndicator extends SMAIndicator {
         series: TLinkedSeries,
         params: RSIParamsOptions
     ): (IndicatorValuesObject<TLinkedSeries>|undefined) {
+
         var period = (params.period as any),
             xVal: Array<number> = (series.xData as any),
-            yVal: Array<Array<number>> = (series.yData as any),
+            yVal: Array<Array<number>> | Array<number> = (series.yData as any),
```

Comment:
Hmm I know it is a small detail, but it would look more natural if you defined that TS definition in the opposite order
`Array<number> | Array<Array<number>>`.